test-module/omod/src/test/java/org/openmrs/module/web/controller/TestModuleControllerJUnit4Test.java

```diff
@@ -24,6 +26,7 @@
 /**
  * Testing with JUnit 4
  */
+@RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
Same question as above. This was already being done by simply extending `org.openmrs.web.test.BaseModuleWebContextSensitiveTest`. So i dont understand this change.

Reply:
Thanks for the catch, it's not necessary, let me fix that